packit_service/service/events.py

```diff
@@ -114,6 +114,27 @@ class TheJobTriggerType(str, enum.Enum):
     copr_end = "copr_end"
 
 
+class EventType(str, enum.Enum):
+    installation = "InstallationEvent"
+    copr_build = "CoprBuildEvent"
+    koji_build = "KojiBuildEvent"
+    tft_result = "TestingFarmResultsEvent"
+    release = "ReleaseEvent"
+    push_gh = "PushGitHubEvent"
+    pull_request_gh = "PullRequestGithubEvent"
+    pull_request_comment_gh = "PullRequestCommentGithubEvent"
+    issue_comment = "IssueCommentEvent"
+    push_pagure = "PushPagureEvent"
+    pull_request_comment_pg = "PullRequestCommentPagureEvent"
+    pull_request_pg = "PullRequestPagureEvent"
+    pull_request_label_pg = "PullRequestLabelPagureEvent"
+    dist_git_event = "DistGitEvent"
+    issue_comment_gl = "IssueCommentGitlabEvent"
+    merge_request_comment_gl = "MergeRequestCommentGitlabEvent"
+    merge_request_gl = "MergeRequestGitlabEvent"
+    push_gl = "PushGitlabEvent"
```

Comment:
- Remove 'Event', as it's mostly redundant (keep only for dist-git, as that one doesn't sound event-like at all without it :) ).
- avoid unnecessary abbreviations (i.e.: 'pg' is already used in other places to note 'postgres')
- change word order, to sound more natural (i.e. to the question "What type of event is this?" one naturally would answer: "a release" or "a GitLab merge request" and so on).
- capitalise "GitLab" as it should be 

```suggestion
    installation = "Installation"
    copr_build = "CoprBuild"
    koji_build = "KojiBuild"
    tft_result = "TestingFarmResult"
    release = "Release"
    github_push = "GitHubPush"
    github_pull_request = "GitHubPullRequest"
    github_pull_request_comment = "GitHubPullRequestComment"
    issue_comment = "IssueComment"
    pagure_push = "PagurePush"
    pagure_pull_request_comment = "PagurePullRequestComment"
    pagure_pull_request = "PagurePullRequest"
    pagure_pull_request_label = "PagurePullRequestLabel"
    dist_git_event = "DistGitEvent"
    gitlab_issue_comment = "GitLabIssueComment"
    gitlab_merge_request_comment = "GitLabMergeRequestComment"
    merge_request = "GitLabMergeRequest"
    gitlab_push = "GitLabPush"
```

Reply:
You're right, these are better, I can definitely change the suggested enum names, but in case of the enum values, they are the same as the event class names, since it is used when creating dict from event object as:
`d["event_type"] = self.__class__.__name__` and then
` self.event_type = EventType(event.get("event_type"))`

Reply:
> They are the same as the event class names

That's my point. Do we need this enum at all? Cannot we work only with the class name?

Reply:
As a bonus. If we can avoid that enum, we can use things like `if self.event_type == GithubPullRequest.__name__`.

Reply:
I see, didn't realise this. Let's improve the enum names then, if it's not too complicated.

Reply:
I just wanted to avoid comparing it directly like `self.event_type == "GithubPullRequest"`, but this way `if self.event_type == GithubPullRequest.__name__` of comparing looks good, so I can remove the enum then